example/package.json

```diff
@@ -5,8 +5,9 @@
     "build": "pulp browserify --optimise --to example.js"
   },
   "devDependencies": {
-    "pulp": "^9.0.1",
-    "purescript": "^0.10.1",
-    "rimraf": "^2.5.4"
+    "bower": "^1.8.4",
```

Comment:
Adding bower as an explicit local dependency doesn't work too well, since it complains if it's not the most up to date version at all times.

Reply:
ok I can remove. this was when I was struggling to get `yarn run example` working but now it does.